monitoring/mongodb/dashboard.py

```diff
@@ -0,0 +1,1359 @@
+    showPoints="auto",
+    legendDisplayMode="table",
+    legendPlacement="right",
+    legendValues=["mean", "max", "min"],
+)
+
+wt_cache_io = mongodb_timeseries(
+    "Cache I/O",
+    [
+        Target(
+            expr=f'sum(rate(mongodb_ss_wt_cache_bytes_{operation}_cache{{job=~"{Jobs.shard_pattern}", rs_nm=~"$rs_nm"}}[$__rate_interval]))',
+            legendFormat=f"{operation}_cache",
+        )
+        for operation in ["written_from", "read_into"]
+    ],
+    unit="binBps",
+    fillOpacity=0,
+    showPoints="auto",
+)
+
+evictions_app_threads = mongodb_timeseries(
+    "Evictions from application threads",
+    [
+        Target(
+            expr=f'rate(mongodb_ss_wt_cache_modified_pages_evicted_by_application_threads{{job=~"{Jobs.shard_pattern}", rs_state="{RS_STATE_PRIMARY}", rs_nm=~"$rs_nm"}}[$__rate_interval])',
+            legendFormat="{{pod}}",
+        )
+    ],
+    unit=UNITS.OPS_PER_SEC,
+    fillOpacity=0,
+    showPoints="auto",
+)
+
+wiredtiger_cache = mongodb_timeseries(
+    "Cache Size",
+    [
+        Target(
+            expr=f'sum({metric}{{job=~"{Jobs.shard_pattern}"}}) by ()',
+            legendFormat=legend,
+        )
+        for metric, legend in [
+            ("mongodb_mongod_wiredtiger_cache_max_bytes", "max"),
+            ("mongodb_mongod_wiredtiger_cache_bytes", "used"),
+            ("mongodb_ss_wt_cache_tracked_dirty_bytes_in_the_cache", "dirty"),
+        ]
+    ],
+    unit=UNITS.BYTES,
+)
+
+# Create the dashboard
+dashboard = (
+    Dashboard(
+        title="MongoDB",
+        editable=True,
+        sharedCrosshair=True,
+        refresh="",
+        tags=[],
+        timezone="browser",
+        inputs=[
+            DataSourceInput(
+                name="DS_PROMETHEUS",
+                label="Prometheus",
+                pluginId="prometheus",
+                pluginName="Prometheus",
+            ),
+        ],
+        templating=Templating(
+            [
+                Template(
+                    name="interval",
+                    label="Interval",
+                    query="1s,5s,1m,5m,1h,6h,1d",
+                    auto=True,
+                    autoCount=200,
+                    autoMin="1s",
+                    includeAll=False,
+                    multi=False,
+                    type="interval",
+                ),
+                Template(
+                    dataSource=DATASOURCE,
+                    name="rs_nm",
+                    label="Instance",
+                    query="label_values(mongodb_mongod_replset_my_state,set)",
+                    includeAll=True,
+                    multi=True,
+                    refresh=1,
+                    type="query",
+                ),
+                Template(
+                    dataSource=DATASOURCE,
+                    name="node_name",
+                    label="Node Name",
+                    query='label_values(mongodb_up{cluster=~"$cluster",service_name=~"$service_name"}, node_name)',
```

Comment:
`cluster` and `service_name` are neither inputs nor "earlier" template --> will not work.
Anyway this template param does not seem to be used (`node_name` is not used in the file, and the template is hidden)